engine-comparison/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM gcr.io/fuzzer-test-suite/clang-deps-image
+
+RUN apt-get update
+
+# Create directory, and set it as the working directory "."
+WORKDIR /~/work
```

Comment:
The default Docker Ubuntu requires `sudo` to modify the root `/`. This directory named `/~` is the 'home' that usr has permissions on